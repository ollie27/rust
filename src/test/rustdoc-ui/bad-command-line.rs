// compile-flags:--does-not-exist
