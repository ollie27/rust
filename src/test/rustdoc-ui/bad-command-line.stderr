error: Unrecognized option: 'does-not-exist'

